### PR TITLE
Add CRUD for Content

### PR DESCRIPTION
**Pull request by twitu**
Deleting Content also deletes all related Steps.

### CODE REVIEW

**Comment by dusnat** [comment]
There should not be an option to delete content. Since content can be shared across different pathways (not all of which have the same owner), deleting the content node would affect all those pathways unexpectedly.

Here's what I suggest as an alternative - 

Delete only steps. Whenever deleting a step, get the content node from that step and see how many edges that node has. If it has zero edges, only then delete the content node (since then we can be sure that nobody else is using the same content node).

**Comment by twitu** [comment]
https://github.com/dush-t/pathways/wiki/Seeding-the-database
Wiki page for seeding database using updated queries.

File: server/src/db/queries/pathway/createUpdatePathway.js [hunk 1]
```diff
@@ -2,54 +2,53 @@ const Query = require('../../core/Query')
 const Model = require('../../core/Model')
 
 const createUpdatePathway = ({id, name, steps}) => {
-    // create id for path if doesn't exist
-    if (!id) {
```
Comment (hedonhermdev) [inline server/src/db/queries/pathway/createUpdatePathway.js:6]: This is unnecessary is ID is a mandatory field right?

**Review by hedonhermdev — APPROVED** [review]

File: server/src/db/queries/pathway/createUpdatePathway.js (hunk 1 above)
Comment (twitu) [reply server/src/db/queries/pathway/createUpdatePathway.js:6]: Yeah id is mandatory. That's why I've removed it. The query will not reach here without and id value.

Comment (hedonhermdev) [reply server/src/db/queries/pathway/createUpdatePathway.js:6]: Sorry, my bad

**Comment by hedonhermdev** [comment]
In what situation are we gonna call `deleteContent`?
The users will not be able to call it right?

**Comment by twitu** [comment]
Delete content is a cleanup/maintenance operation, to remove unrelated content that has not been used for a long time. It can be manual or automated. There are two ways to go about it,

1. A private API implemented using directives that only allow admins to use it.
2. Directly run cypher queries from the db interface

Either way it's not the most relevant now. Can be added later.

**Pull request merged**